composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "netzstrategen/image-disclaimer",
+    "type": "wordpress-plugin",
+    "authors": [
+        {
+            "name": "netzstrategen",
+            "email": "hallo@netzstrategen.com",
+            "homepage": "https://netzstrategen.com"
+        }
+    ],
+    "dist": {
+        "url": "git@github.com:netzstrategen/woocommerce-image-disclaimer.git",
+        "type": "git"
+    },
+    "require": {
+        "php": ">=7.0.0",
+        "composer/installers": "^1"
```

Comment:
Please remove; this is a requirement of the root project that wants to use this plugin, but not of this plugin.